### ISSUE TITLE
Truncate removes "unique" field in index ops

### ISSUE THREAD

**Issue by funny-falcon** [tarantool/tarantool#2060]
Reported by @lenkis to tarantool/go-tarantool#32 , but real issue is in tarantool itself.
version 1.7.3-196-g92133b2

1. Begin with a clean empty database.
Start the server and execute this script,
which is the same as what's in the Connectors chapter:

```lua
box.cfg{listen=3301}
box.schema.space.create('examples',{id=999})
box.space.examples:create_index('primary', {type = 'hash', parts = {1, 
'unsigned'}})
box.schema.user.grant('guest','read,write','space','examples')
box.schema.user.grant('guest','read','space','_space')
local console = require 'console'
console.listen '0.0.0.0:3302'
```

2. connect to admin console, fetch information for index before and after `truncate`:
```
$ rlwrap nc 0 3302
Tarantool 1.7.3 (Lua console)                                  
type 'help' for interactive help                               
box.space._index:get({999,0})
---
- [999, 0, 'primary', 'hash', {'unique': true, 'lsn': 1}, [[0, 'unsigned']]]
...
box.space._vindex:get({999,0})
---
- [999, 0, 'primary', 'hash', {'unique': true, 'lsn': 1}, [[0, 'unsigned']]]
...
box.space.examples:truncate()
---
...
box.space._index:get({999,0})
---
- [999, 0, 'primary', 'hash', {'lsn': 5}, [[0, 'unsigned']]]
...
box.space._vindex:get({999,0})
---
- [999, 0, 'primary', 'hash', {'lsn': 5}, [[0, 'unsigned']]]
...
```

**Comment by funny-falcon** [comment]
Любопытно, что unique=false сохраняется.

**Comment by rtsisyk** [comment]
Fixed by 5aee8a104a1369c9eaf7673ce3d7e11ebdb75d36

**Comment by rtsisyk** [comment]
Discussed that we will implemented truncate via `space_version` in space options.
